recipes/python-rocksdb/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "python-rocksdb" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa692fbd06d8ed4ad641feee2970062fe01120e87b645a29403c71c628639e4f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  missing_dso_whitelist:
+    - '*/libc.so.6'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - setuptools >=25
+    - rocksdb >=6.10.1
```

This is not yet listed in the pinning, so I made a PR: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/654

Also remove the number so that it gets picked up by the global pinning (which intially be `6.10.2`)

```suggestion
    - rocksdb
```